paper/paper.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "CDP: Software for Municipal Data Collection, Analysis, and Publication"
+tags:
+    - Python
+    - JavaScript
+    - open government
+    - open data
+    - open infrastructure
+    - municipal governance
+    - data archival
+    - civic technology
+    - natural language processing
+authors:
+    - name: Jackson Maxfield Brown
+      orcid: 0000-0003-2564-0373
+      affiliation: 1
+    - name: To Huynh
+      orcid: 0000-0002-9664-3662
+      affiliation: 2
+    - name: Isaac Na
+      orcid: 0000-0002-0182-1615
+      affiliation: 3
+    - name: Brian Ledbetter
+      affiliation: 1
+    - name: Hawk Ticehurst
+      affiliation: 1
+    - name: Sara Liu
+      affiliation: 4
+    - name: Emily Giles
+      affiliation: 4
+    - name: Katlyn Greene
+      affiliation: 4
+    - name: Sung Cho
+      affiliation: 4
+    - name: Shak Ragoler
+      affiliation: 4
+    - name: Dhanya Pisharasyar
+      affiliation: 4
+    - name: Nicholas Weber
+      orcid: 0000-0002-6008-3763
+      affiliation: 1
+
+affiliations:
+    - name: University of Washington iSchool, University of Washington, Seattle
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: McKelvey School of Engineering, Washington University, St. Louis
+      index: 3
+    - name: Independent Contributor
+      index: 4
+
+date: 29 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Cities, counties, and states throughout the USA are bound by law to archive recordings of public meetings. Most local governments comply with these laws by posting documents, audio, or video recording online. Council Data Project (CDP) is three open-source tools that improve the accessibility of local government data by systematically collecting, transforming, and re-publishing this data to the web. The software provided here can be used by civic technologists to promote government transparency, or in support of research by using processed data for public policy, natural language processing and machine learning, or information retrieval and discovery.
+
+# Statement of Need
+
+Comparative research into municipal governance in the USA is often prohibitively difficult due to a broad federal system where states, counties, and cities divide legislative powers differntly. This has contributed to the lack of large-scale quantitative studies of municipal government, and impeded necessary research into  effective procedural elements of administrative and legislative processes [@Trounstine:2009]. Council Data Project (CDP) enables large-scale quantitative studies by generating standardized municipal governance corpora - inclduing X, Y, and Z.
```

Comment:
The XYZ here is to put examples (legsitlative voting records, time-stamped transcripts with speaker annotations, n-gram tokens, etc)